ruby-challenges.rb

```diff
@@ -31,3 +53,20 @@
 # Expected output: 'LEARN is not a palindrome'
 palindrome_tester3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+def palindrome_tester (str) 
+    if str = str.downcase.split.reverse
+        return true
+        #puts "#{str} is a palindrome"
+    elsif
+        return false
+        #puts "#{str} is not a palindrome"
+     end
+   end
```

Comment:
Logic was close on this.  Keep in mind that the single equal sign will assign a value to a variable.  A conditional needs to check equality which will need a double equal sign.  Also, you will want to downcase both the initial string and the reversed string as well so that it is case insensitive.